examples/mplot3d/trisurf3d_demo2.py

```diff
@@ -1,11 +1,28 @@
+'''
+Two additional examples of plotting surfaces with triangular mesh.
+
+The first demonstrates use of plot_trisurf's triangles argument, and the
+second sets a Triangulation object's mask and passes the object directly
+to plot_trisurf.
+'''
+
 import numpy as np
 import matplotlib.pyplot as plt
 from mpl_toolkits.mplot3d import Axes3D
 import matplotlib.tri as mtri
 
-# u, v are parameterisation variables
-u = (np.linspace(0, 2.0 * np.pi, endpoint=True, num=50) * np.ones((10, 1))).flatten()
-v = np.repeat(np.linspace(-0.5, 0.5, endpoint=True, num=10), repeats=50).flatten()
+
+fig = plt.figure(figsize=plt.figaspect(0.5))
```

Comment:
I made the two plots in this example display side by side as subplots.  Previously the code was only actually displaying one of them when run.